recipes/sparsehash/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.0.3" %}
+{% set checksum = "05e986a5c7327796dad742182b2d10805a8d4f511ad090da0490f146c1ff7a8c" %}
+{% set checktype = "sha256" %}
+
+package:
+    name: sparsehash
+    version: {{ version }}
+
+source:
+    fn: sparsehash-{{ version }}.tar.gz
+    url: https://github.com/sparsehash/sparsehash/archive/sparsehash-{{ version }}.tar.gz
+    {{ checktype }}: {{ checksum }}
+
+build:
+    number: 0
+    skip: true  # [win]
+
+requirements:
+    build:
+        - pkg-config # [not win]
+    run:
+        - pkg-config # [not win]
+
+test:
+    commands:
+        - test -f ${PREFIX}/lib/pkgconfig/libsparsehash.pc # [not win]
```

Comment:
Are there any libraries or includes we can check for? Please add them here too. Also please drop the selector as it is unneeded.